pylibczi/tests/czi_scene_oldtests.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+from pylibczi import CziScene
+from pylibczi.exceptions import CziFileNotMultiSceneError
+
+
+@pytest.mark.parametrize("fname, xp_query, expected", [
+#    pytest.param('s_1_t_1_c_1_z_1.czi', None, None, marks=pytest.mark.raises(exception=CziFileNotMultiSceneError)),
```

Comment:
Remove these tests? Or fix?

Reply:
fixing is better... also, it's kinda impolite to put large files in the repo just for testing, right?